README.md

```diff
@@ -72,16 +74,19 @@ Download stable releases of the standalone capa binaries [here](https://github.c
 
 To use capa as a library or integrate with another tool, see [doc/installation.md](https://github.com/mandiant/capa/blob/master/doc/installation.md) for further setup instructions.
 
-For more information about how to use capa, see [doc/usage.md](https://github.com/mandiant/capa/blob/master/doc/usage.md).
+# web explorer
+The [capa Web Explorer](https://mandiant.github.io/capa/#/) enables you to interactively explore capa results in your web browser. Besides the online version you can download a standalone HTML file for local offline usage. More details on the web UI is available in the [capa Web Explorer README](https://github.com/mandiant/capa/blob/master/web/explorer/README.md).
```

Comment:
I'd like to see a screenshot here, especially if it will feature this prominently.

Maybe we want to introduce a table to compare/contrast the ways of consuming capa output, including CLI, IDA, Ghidra, CAPE, web, etc.